crud/common/rebalance.lua

```diff
@@ -0,0 +1,211 @@
+local fiber = require('fiber')
+local log = require('log')
+local vshard_consts = require('vshard.consts')
+local utils = require('crud.common.utils')
+
+local has_metrics_module, metrics = pcall(require, 'metrics')
+
+local SETTINGS_SPACE_NAME = '_crud_settings'
+local SAFE_MOD_ENABLE_EVENT = '_crud.safe_mode_enable'
+
+local M = {
+    safe_mode = false,
+    safe_mode_enable_hooks = {},
+    safe_mode_disable_hooks = {},
+    _router_cache_last_clear_ts = fiber.time()
+}
+
+local function create_space()
```

Comment:
AFAICS, the space has general name `_crud_settings`, its schema is also general. So, I suppose, in the future, we may have new crud settings, which we want to persist. So, the space has nothing to do with `rebalance` module, which just uses the space. And it seems its creation should be done somewhere in more general place, e.g. `crud.schema` or smth like that.

Reply:
Okay. Renamed this space back and moved it's creation to `crud.schema` module.